tools/src/tester/StoryEvaluator.ts

```diff
@@ -46,9 +46,21 @@ export default class StoryEvaluator {
         chapters: []
       }
     }
-    const prologues = await this.#evaluate_supplemental_chapters(this.story.prologues ?? [])
-    const chapters = await this.#evaluate_chapters(this.story.chapters)
-    const epilogues = await this.#evaluate_supplemental_chapters(this.story.epilogues ?? [])
+    const variables_error = check_story_variables(this.story)
+    if (variables_error) {
+      return {
+        result: Result.ERROR,
+        display_path: this.display_path,
+        full_path: this.full_path,
+        description: this.story.description,
+        chapters: [],
+        message: variables_error
+      }
+    }
```

Comment:
These errors should be assigned to the chapters that throw them so the view can pinpoint where to fix.